### DISCUSSION TITLE
Are the docs outdated?

### DISCUSSION THREAD

**Discussion by Manitej66** [nextauthjs/next-auth#7113] [Help] [answered]
i'm currently at this page https://next-auth.js.org/getting-started/client#updating-the-session

i wanna use the `update` session but in the editor it says, 
```
Property 'update' does not exist on type '{ readonly data: null; readonly status: "loading"; } | { data: Session; status: "authenticated"; } | { data: null; status: "loading"; } | { data: Session; status: "authenticated"; } 
```

<img width="961" alt="image" src="https://user-images.githubusercontent.com/61357713/228742661-0efe43ed-eeb4-4edb-8ffc-9d6263905316.png">

**Comment by henriq-88** [comment, marked as answer]
I'm also currently waiting for this feature to be released. [It seems to have been merged into the codebase](https://github.com/nextauthjs/next-auth/pull/7056), but no new version has been published yet.
